lib/srv/forward/sshserver.go

```diff
@@ -682,6 +726,22 @@ func (s *Server) handleChannel(ctx context.Context, nch ssh.NewChannel) {
 	channelType := nch.ChannelType()
 
 	switch channelType {
+	// Channels of type "tracing-request" are sent to determine if ssh tracing envelopes
+	// are supported. Accepting the channel indicates to clients that they may wrap their
+	// ssh payload with tracing context.
+	case tracessh.TracingChannel:
+		ch, _, err := nch.Accept()
+		if err != nil {
```

Comment:
Can we log this error [like in the other channel](https://github.com/gravitational/teleport/blob/217f7240ad966b17abb9e041d4be5b9b6673998c/lib/srv/forward/sshserver.go#L903) types?